frontend/src/App.svelte

```diff
@@ -7,32 +7,20 @@
     Redo,
     ArrowDownToLine,
     ArrowLeftSquare,
-    MoveUpRight,
-    Sparkle,
+    ArrowUpRight,
+    Wand,
   } from "lucide-svelte";
   import { onMount } from "svelte";
   import paper from "paper";
   import { throttle, debounce } from "throttle-debounce";
 
-  import BackgroundGradient from "$lib/BackgroundGradient.svelte";
   import modalLogoWithText from "$lib/assets/logotype.svg";
   import Paint from "$lib/Paint.svelte";
   import easterEggImage from "$lib/assets/mocha_outside.png";
   import valleyImg from "$lib/assets/valley.png";
   import turboArtTitleGif from "$lib/assets/turbo-art-title.gif";
-  import resolveConfig from "tailwindcss/resolveConfig";
-  import tailwindConfig from "../tailwind.config.js";
   import PreviewImages from "$lib/PreviewImages.svelte";
 
-  const fullConfig = resolveConfig(tailwindConfig);
-  const breakpointSm = parseInt(fullConfig.theme.screens.sm);
-  $: isMobile = false;
-
-  const handleWindowResize = () => {
-    const isSmallWindow = window.innerWidth <= breakpointSm;
-    isMobile = isSmallWindow;
-  };
```

Comment:
🤣 man I was a noob, thanks for cleaning up!

Reply:
I see why you did this but it's simpler with the tw bps!